<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>classificationWindow</class>
 <widget class="QDialog" name="classificationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>733</y>
     <width>176</width>
     <height>62</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2"/>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>561</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>分类方案选择 </string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>文件统计面板</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>411</width>
     <height>701</height>
    </rect>
   </property>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>411</width>
      <height>701</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>409</width>
       <height>699</height>
      </rect>
     </property>
     <widget class="QLabel" name="totalFileCountLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>401</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>文件总数：</string>
      </property>
     </widget>
     <widget class="QLabel" name="pathLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>401</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>路径：</string>
      </property>
     </widget>
     <widget class="QChartView" name="fileTypeChartView" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>391</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>670</y>
        <width>391</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>注：不包含只读/系统/隐藏文件及文件夹</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>561</width>
     <height>701</height>
    </rect>
   </property>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>561</width>
      <height>701</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>559</width>
       <height>699</height>
      </rect>
     </property>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>541</width>
        <height>150</height>
       </rect>
      </property>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>521</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>占比小于5%的文件归类为“其他”（不勾选则不予处理）</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>按文件类型分类</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>511</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>占比小于5%的文件也分别单独进行归类（不勾选则不予处理）</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="volumeWidget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>541</width>
        <height>150</height>
       </rect>
      </property>
      <widget class="QPushButton" name="pushButton_size">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>按文件体积分类</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>50</y>
         <width>251</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>小于                                                        </string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>381</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>181</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>小于                                                                  </string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>50</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="maximum">
        <double>9999.989999999999782</double>
       </property>
       <property name="singleStep">
        <double>10.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="maximum">
        <double>9999.989999999999782</double>
       </property>
       <property name="singleStep">
        <double>10.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>80</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="maximum">
        <double>9999.989999999999782</double>
       </property>
       <property name="singleStep">
        <double>10.000000000000000</double>
       </property>
       <property name="value">
        <double>100.000000000000000</double>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>461</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>大于                                                               </string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>110</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="maximum">
        <double>9999.989999999999782</double>
       </property>
       <property name="singleStep">
        <double>50.000000000000000</double>
       </property>
       <property name="value">
        <double>100.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>371</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>注：范围如有重叠，重叠范围内的文件将会被复制多份</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>50</y>
         <width>40</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>K</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>50</y>
         <width>40</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>M</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>80</y>
         <width>40</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>M</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>80</y>
         <width>40</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>M ——</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>110</y>
         <width>40</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>M</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <double>9999.989999999999782</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_6" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>541</width>
        <height>151</height>
       </rect>
      </property>
      <widget class="QPushButton" name="pushButton_time">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>按修改时间分类</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>331</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>                                   天内</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>7</number>
       </property>
       <property name="displayIntegerBase">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>381</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>注：范围如有重叠，重叠范围内的文件将会被复制多份</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>301</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>                                   月内</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>301</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>                                   年内</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>361</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft PhagsPa</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>设置好后 点击按钮查看分类效果预览</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>740</y>
     <width>981</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::StrongFocus</enum>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>QtCharts/QChartViewh</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
